nss-day-cohort-56/rare-server-team-1-2
# Cml post view

# Description

built create retrieve update and destroy functions for post 
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ -] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
to test
fetch origin cml-PostView\
cd there
migrate for changes to post model 
load data for post and tags and categories
run server
go to dom and look for a list of all post 
click on post title to see details
click edit post to edit 
delete a post 
create a new post by going to the new post tab


# Checklist:
